src/main/kotlin/org/ozwillo/dcimporter/web/Routes.kt

```diff
@@ -75,6 +74,7 @@ class Routes(
             PUT("/type/{type}", datacoreHandler::updateResourceWithOrganization)
             GET("/organizations", datacoreHandler::getAllOrganization)
             GET("/models", datacoreHandler::getModels)
+            GET("/model/{project}/{type}", datacoreHandler::getModel)
```

Comment:
Is project really neeeded in the URI ? For others handlers the project value is define in back. And I think that we can use the mother project oasis.main to deal with all DC models.